README.md

```diff
@@ -48,37 +48,17 @@ and one of {`jackhmmer`, [MMseqs2](https://github.com/soedinglab/mmseqs2) (night
 installed on on your system. You'll need `git-lfs` to download OpenFold parameters. 
 Finally, some download scripts require `aria2c` and `aws`.
 
-For convenience, we provide a script that installs Miniconda locally, creates a 
-`conda` virtual environment, installs all Python dependencies, and downloads
-useful resources, including both sets of model parameters. Run:
+This package is currently supported for CUDA 11 and Pytorch 1.12
 
-```bash
-scripts/install_third_party_dependencies.sh
-```
-
-To activate the environment, run:
-
-```bash
-source scripts/activate_conda_env.sh
```

Comment:
I guess we shouldn't remove these since people will have the old env, but it's kind of confusing that they refer to the old setup

Reply:
If we want to keep information/scripts about the old set up, my inclination would be to have a separate section / README for them. These scripts are also more useful if someone is running from a slurm job, so maybe I can move these scripts there and provide some instructions?